<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiplayerWindow</class>
 <widget class="QDialog" name="MultiplayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonCanon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Canon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonArcherTower">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Archer Tower</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonWizardTower">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>440</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Wizard Tower</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChooseWeapon">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>                     Choose Weapon</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCanonPic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Canon Pic</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelArcherTowerPic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Archer Tower Pic</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWizardTowerPic">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wizard Tower Pic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButtonCanon</zorder>
  <zorder>pushButtonArcherTower</zorder>
  <zorder>pushButtonWizardTower</zorder>
  <zorder>labelChooseWeapon</zorder>
  <zorder>labelCanonPic</zorder>
  <zorder>labelArcherTowerPic</zorder>
  <zorder>labelWizardTowerPic</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
